src/pages/home/HomePage.tsx

```
@@ -1,19 +1,20 @@
-        backgroundColor: '#BDBDBD',
-      }}
-    >
-      <TextField fieldSize="small" placeholder="YYYY" maxLength={4} />
-      <TextField fieldSize="large" placeholder="이름을 입력해주세요." />
-      <TextField fieldSize="thin" placeholder="ex) 솝트특별자치시 앱잼구" />
+    <div>
+      <h1>Home Page</h1>
+      <TokenRefreshTest />
+      <LogoutButton />
+      <br />
+      <button onClick={() => navigate('/signup')}>Signup</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)